Ansible/todoapp-playbook.yml

```diff
@@ -0,0 +1,75 @@
+- name: Run Todo App
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key 
+    prompt: Enter your trello api key?
+    private: no
```

Comment:
This is fine here (it makes typing easier!) but if this were a real project we'd probably want this as yes, even if we were only using it locally for now - at some point it's likely that this will end up running somewhere more centrally (e.g. in a pipeline) and at that stage we won't want our secrets potentially being exposed